network/definition/data/components/vcrear/vcrear-rx.yaml

```diff
@@ -12,6 +12,19 @@ messages:
   BMSB_currentLimit:
     destBuses: ass
     sourceBuses: veh
+  PM100DX_temps1:
+    destBuses: veh
+    sourceBuses: ass
```

Comment:
@JoshLafleur why does this need to be specified in the rx file? We should be able to infer this on the rx side, unless there are multiple possible paths (in which case we should pick the shortest one by default unless manually specified)